src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -921,6 +921,9 @@ def visit_span(self, span: Span) -> None:
         if not description:
             return
 
+        if description[:3].upper() != "GET":
```

Comment:
I saw in some of the test fixtures we have queries that start with newline characters or whitespace, should we also call strip on the description to get around cases like that?

Reply:
Wouldn't hurt! Added